l2geth/eth/api_backend.go

```diff
@@ -400,6 +401,10 @@ func (b *EthAPIBackend) RPCGasCap() *big.Int {
 	return b.eth.config.RPCGasCap
 }
 
+func (b *EthAPIBackend) RPCEVMTimeout() time.Duration {
```

Comment:
Adding the `0 == infinite` note to a doc comment here would be helpful

Reply:
no docs are on any of the other methods do you still want one here?

Reply:
I added the (0=infinite) to the docs for the interface